~/.kube/config

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
are you sure you want to add kubeconfig?

Reply:
interesting that the redhat pre-commit lets me commit this; I guess they haven't added a pattern for this yet

Reply:
bummer